client/me/purchases/billing-history/receipt.jsx

```diff
@@ -190,6 +194,89 @@ function ReceiptPaymentMethod( { transaction } ) {
 	);
 }
 
+function VatDetails( { transaction } ) {
+	const translate = useTranslate();
+	const { vatDetails, isLoading, fetchError } = useVatDetails();
+	const reduxDispatch = useDispatch();
+
+	const getEmailReceiptLinkClickHandler = ( receiptId ) => {
+		return ( event ) => {
+			event.preventDefault();
+			reduxDispatch( recordGoogleEvent( 'Me', 'Clicked on Receipt Email Button' ) );
+			reduxDispatch( sendBillingReceiptEmail( receiptId ) );
+		};
+	};
+
+	if ( isLoading || fetchError || ! vatDetails.id ) {
+		return null;
+	}
+
+	return (
+		<>
+			<li>
+				<strong>{ translate( 'VAT Details' ) }</strong>
+				<span className="receipt__vat-vendor-details-description">
+					{ translate(
+						'{{noPrint}}You can edit your VAT details {{vatDetailsLink}}on this page{{/vatDetailsLink}}. {{/noPrint}}This is not an official VAT receipt. For an official VAT receipt, {{emailReceiptLink}}email yourself a copy{{/emailReceiptLink}}.',
+						{
+							components: {
+								noPrint: <span className="receipt__no-print" />,
+								vatDetailsLink: <a href={ vatDetailsPath } />,
+								emailReceiptLink: (
+									<Button
+										plain
+										className="receipt__email-button"
+										onClick={ getEmailReceiptLinkClickHandler( transaction.id ) }
+									/>
+								),
+							},
+						}
+					) }
+				</span>
+				{ vatDetails.name }
+				<br />
+				{ vatDetails.address }
+				<br />
+				{ translate( 'VAT #: %(vatCountry)s %(vatId)s', {
+					args: {
+						vatCountry: vatDetails.country,
+						vatId: vatDetails.id,
+					},
+					comment: 'This is the user-supplied VAT number, format "UK 553557881".',
+				} ) }
+			</li>
+			<li>
+				<strong>{ translate( 'Vendor VAT Details' ) }</strong>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **38** times:
`translate( 'Vendor VAT #:' )` *ES Score: 6*